src/scss/components/fat-footer/_footer--column-one.scss

```diff
@@ -2,7 +2,6 @@
 
 /* First Column Styles */
 .su-local-footer {
-  // logo
   .su-fat-footer__logo {
     max-width: 250px;
     @include modular-spacing('padding-bottom', 4);
```

Comment:
Feel free to do this your way or our usual way (see the other comment). Same goes for the other ones similar to this.